### ISSUE TITLE
Org-Protocol for org-chef?

### ISSUE THREAD

**Issue by MElemental** [Chobbes/org-chef#53]
Perhaps someone could make an org-protocol browser extension or bookmarklet for Org-chef.
This would save a step on the capturing process. Org-roam uses org-protocol.

Essentially this would send a capture call to emacs with the URL of the website.
It's one less step than copying the URL and pasting it into emacs.
I'm not too familiar with javascript, org-protocol, or org-chef to write the code snippet, but it seems relatively simple to write a bookmarklet snippet with the org-protocol call to emacs.

https://orgmode.org/worg/org-contrib/org-protocol.html

**Comment by Chobbes** [comment]
Yeah, sure, that would be cool! [Here it is.](https://github.com/Chobbes/org-chef?tab=readme-ov-file#org-protocol)

Should work with the latest version of org-chef no problem :).
